docs/data/data-grid/demo/PopularFeaturesDemo.tsx

```diff
@@ -0,0 +1,383 @@
+    plan: 'Pro',
+    detailPage: '/row-pinning/',
+    demo: <RowPinningWithPagination />,
+    newBadge: true,
+  },
+  {
+    id: 12,
+    name: 'Aggretation and Summary rows',
+    description: 'Set summary footer rows or inline summaries with row grouping',
+    plan: 'Premium',
+    detailPage: '/aggregation/',
+    demo: <AggregationRowGrouping />,
+    newBadge: true,
+  },
+  {
+    id: 13,
+    name: 'Column Visibility',
+    description:
+      'Display different columns in different use cases by defining which columns are visible',
+    plan: 'Community',
+    detailPage: '/column-visibility/',
+    demo: <ColumnSelectorGrid />,
+  },
+  {
+    id: 14,
+    name: 'Column Virtualization',
+    description: 'High performance support for thousands of columns',
+    plan: 'Community',
+    detailPage: '/virtualization/#column-virtualization',
+    demo: <ColumnVirtualizationGrid />,
+  },
+  {
+    id: 15,
+    name: 'Row Virtualization',
+    description: 'High performance support for vast volume of data',
+    plan: 'Pro',
+    detailPage: '/virtualization/#row-virtualization',
+    demo: <FullFeaturedDemo />,
+  },
+  {
+    id: 16,
+    name: 'Tree data',
+    description: 'Support rows with parent / child relationship',
+    plan: 'Pro',
+    detailPage: '/tree-data/',
+    demo: <TreeDataFullExample />,
+  },
+];
+
+const getChipProperties = (plan: string) => {
+  switch (plan.toLowerCase()) {
+    case 'premium':
+      return { avatarLink: '/static/x/premium.svg', color: '#ffecc8' };
+    case 'pro':
+      return { avatarLink: '/static/x/pro.svg', color: '#c8e9ff' };
+    default:
+      return { avatarLink: undefined, color: '#c8ffdb' };
+  }
+};
+
+const PlanTag = (props: { plan: string }) => {
+  const chipPropperties = getChipProperties(props.plan);
+  const avatar = !chipPropperties.avatarLink ? undefined : (
+    <Avatar src={chipPropperties.avatarLink} />
+  );
+  return (
+    <Chip
+      avatar={avatar}
+      sx={{ background: chipPropperties.color, color: 'rgba(0, 0, 0, 0.87)' }}
+      label={props.plan}
+    />
+  );
+};
+
+const CustomToolbar = () => {
+  return (
+    <GridToolbarContainer sx={{ p: 1 }}>
+      <GridToolbarQuickFilter />
+    </GridToolbarContainer>
+  );
+};
+
+const RowDemo = (props: { row: Row }) => {
+  const { row } = props;
+  const theme = useTheme();
+  const gridBgColor = theme.palette.mode === 'dark' ? '#000' : '#fff';
+  const panelColor = theme.palette.mode === 'dark' ? 'transparent' : '#efefef';
+
+  return (
+    <Box sx={{ py: 2, background: panelColor }}>
+      <Box style={{ width: '90%', margin: 'auto', background: gridBgColor }}>
+        {row.demo}
+      </Box>
+    </Box>
+  );
+};
+
+const columns: GridColDef[] = [
+  {
+    field: 'name',
+    headerName: 'Feature name',
+    maxWidth: 160,
+    flex: 0.2,
+    minWidth: 100,
+    groupable: false,
+    renderCell: (params) => {
+      if (!params.value) {
+        return <React.Fragment />;
```

Comment:
```suggestion
        return '';
```

Usually we uses `""` in our codebase to return nothing but avoid falling back to `params.formattedValue`